pallets/asset-index/src/lib.rs

```diff
@@ -450,8 +514,48 @@ pub mod pallet {
             Ok(())
         }
 
-        fn remove_asset(_: &T::AssetId) -> DispatchResult {
-            todo!();
+        /// Remove asset with provied units.
+        ///
+        /// * If is liquid asset, transfer the ownership to the provied recipient
+        /// * Remove asset with provied units directly if is SAFT
+        fn remove_asset(
+            asset_id: &T::AssetId,
+            units: &T::Balance,
+            recipient: Option<MultiLocation>,
+        ) -> DispatchResult {
+            // TODO:
+            //
+            // Should we force check if AssetId and recipient are paired?
```

Comment:
can you elaborate on this?

Reply:
For example, both `Some` or both `None`, this is because we only have a `remove_assset`